### PR TITLE
Several improvements

### PR DESCRIPTION
**Pull request by VasiliyMatutin**

### CODE REVIEW

**Comment by VasiliyMatutin** [comment]
Alina, please verify on overlooked .idea project files

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -10,14 +9,14 @@
     android:screenOrientation="portrait">
 
     <TextView
-        android:id="@+id/source_language_request_text"
```
Comment (ascherdakova) [inline app/src/main/res/layout/activity_main.xml]: What is the purpose? I don't quit get...

File: build.gradle [hunk 2]
```diff
@@ -6,7 +6,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0'
```
Comment (ascherdakova) [inline build.gradle]: Is this change necessary for some lib?

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (VasiliyMatutin) [reply app/src/main/res/layout/activity_main.xml]: This is usability feature. Just change text field. Because it is more reasonable that user sees target language first on his device system language, while target languages depend on source and available on corresponding language

File: build.gradle (hunk 2 above)
Comment (VasiliyMatutin) [reply build.gradle]: This connected with new version of Android Studio 3.1.1. I think we should support its actual state
